specification/ai/data-plane/Face/stable/v1.2/Face.json

```diff
@@ -0,0 +1,7112 @@
+        "summary": "Delete all session related information for matching the specified session id.",
+        "description": "> [!NOTE]\n> Deleting a session deactivates the Session Auth Token by blocking future API calls made with that Auth Token. While this can be used to remove any access for that token, those requests will still count towards overall resource rate limits. It's best to leverage TokenTTL to limit length of tokens in the case that it is misused.",
+        "parameters": [
+          {
+            "name": "sessionId",
+            "in": "path",
+            "description": "The unique ID to reference this session.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully deleted session and all correlated data."
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/FaceErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Liveness Session": {
+            "$ref": "./examples/LivenessSessionOperations_DeleteLivenessSession.json"
+          }
+        }
+      }
+    },
+    "/detectLivenessWithVerify-sessions": {
+      "post": {
+        "operationId": "LivenessSessionOperations_CreateLivenessWithVerifySession",
+        "summary": "Create a new liveness session with verify. Provide the verify image during session creation.",
+        "description": "A session is best for client device scenarios where developers want to authorize a client device to perform only a liveness detection without granting full access to their resource. Created sessions have a limited life span and only authorize clients to perform the desired action before access is expired.\n\nPermissions includes...\n>\n*\n  * Ability to call /detectLivenessWithVerify/singleModal for up to 3 retries.\n  * A token lifetime of 10 minutes.\n\n> [!NOTE]\n>\n> *\n>   * Client access can be revoked by deleting the session using the Delete Liveness With Verify Session operation.\n>   * To retrieve a result, use the Get Liveness With Verify Session.\n>   * To audit the individual requests that a client has made to your resource, use the List Liveness With Verify Session Audit Entries.\n\nRecommended Option: VerifyImage is provided during session creation.",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "name": "Parameters",
+            "in": "formData",
+            "description": "The parameters for creating session.",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Do you mean expanding each field in `CreateLivenessWithVerifySessionJsonContent` as a part in MFD?
Something looks like
```
        "parameters": [
          {
            "name": "livenessOperationMode",
            "in": "formData",
            "description": "Type of liveness mode the client should follow.",
            "required": true,
            "type": "string"
          },
          {
            "name": "deviceCorrelationIdSetInClient",
            "in": "formData",
            "description": "Type of liveness mode the client should follow.",
            "required": true,
            "type": "bool"
          },
...
          {
            "name": "VerifyImage",
            "in": "formData",
            "description": "The image stream for verify. Content-Disposition header field for this part must have filename.",
            "required": true,
            "type": "file"
          }
        ],
```
This would be different from the current design. We put all fields inside json and serialize them into a string, and put them inside one part in MFD. I think there is no way to describe current design schema completely in swagger 2.0.

Reply:
That is what I meant, and when you say
> We put all fields inside json and serialize them into a string

I think you mean "We require the user to put all the fields into an object and serialize that to JSON". AND the REST API gives _no_ indication of this or what fields belong there. So a user trying to use the REST API would be completely stymied.

Reply:
Make sense to me.
BTW, should we also change the name "VerifyImage" to camelCase?

Reply:
Camel case is probably best, as that is what we recommend for JSON. But we don't have formal guidance on field names in multipart/form-data so I think there is wiggle room here.

Reply:
Updated